src/Utils/RestaurantStats.php

```diff
@@ -523,6 +524,8 @@ public function getRowValue($column, $index, $formatted = true)
                 return $this->formatNumber($order->getRefundTotal(), !$formatted);
             case 'net_revenue':
                 return $this->formatNumber($order->getRevenue(), !$formatted);
+            case 'nonprofit':
+                return $this->formatNumber($order->getNonprofit(), !$formatted);
```

Comment:
`$order->getNonprofit()` will return an instance of `Nonprofit`, so it will break when using `$this->formatNumber`

It should be `$order->getNonprofit()->getName()`